sdp/run_processors.py

```diff
@@ -42,6 +44,22 @@
 logger.addHandler(handler)
 logger.propagate = False
 
+def update_processor_imports(config_path: str, init_file: str = None):
+    """
+    Update processor imports based on config file.
+    
+    Args:
+        config_path: Path to the YAML config file
+        init_file: Optional path to __init__.py file to update
+    """
+    try:
+        manager = ImportManager()
+        manager.sync_with_config(config_path, init_file)
+        logger.info(f"Successfully updated imports for config: {config_path}")
+    except Exception as e:
```

Comment:
Let's avoid using general `Exception`. Could we use a more specific exception type here to make error handling clearer?